onnxruntime/test/providers/nnapi/nnapi_basic_test.cc

```diff
@@ -388,26 +408,135 @@ TEST(NnapiExecutionProviderTest, TestQDQReshape) {
 TEST(NnapiExecutionProviderTest, TestQDQSoftMax) {
   RunQDQModelTest(BuildQDQSoftMaxTestCase<uint8_t, uint8_t>(
                       {1, 32} /* input_shape */,
-                      static_cast<int64_t>(1) /* axis */),
+                      static_cast<int64_t>(1) /* axis */,
+                      1.f / 256 /* output_scales */,
+                      0 /* output_zp */),
                   "nnapi_qdq_test_graph_softmax",
                   {
                       true /* verify_entire_graph_use_ep */
                   });
 }
 
+#if !defined(__ANDROID__)
+// See https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc#L1236
```

Comment:
if using github links, better to use a more permanent one like `https://github.com/microsoft/onnxruntime/blob/fde847473b91c7feeaefc1ee641306a8147fcf0b/onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc#L1236` otherwise line numbers could become outdated.
here, a comment about what the expectation is might be simpler